spec/haskell/src/SEL4/Object/ObjectType/RISCV64.hs

```diff
@@ -217,5 +217,8 @@ capUntypedSize (ASIDPoolCap {}) = bit asidPoolBits
 prepareThreadDelete :: PPtr TCB -> Kernel ()
 prepareThreadDelete _ = return ()
 
+-- No arch-specific operations needed before changing the domain of a TCB on
+-- RISC-V platform.
```

Comment:
suggest dropping platform here, as it's an architecture, platforms are individual boards

Reply:
Should I update the other comment that I copied for this as well?